src/components/motebehov/UtdragFraSykefravaeret.tsx

```diff
@@ -105,21 +105,22 @@ export const UtvidbarTittel = (utvidbarTittelProps: UtvidbarTittelProps) => {
   const showPapirLabel = sykmelding.papirsykmelding;
   return (
     <div className="utdragFraSykefravaeret__utvidbarTittel">
-      <div>
+      <div style={{ display: "flex", flex: 1, justifyContent: "flex-start" }}>
```

Comment:
Styled-components her for å separere presentasjon og styling?

Reply:
Ooops, denne har jeg oversett. Jeg lager en styled component